src/frontend/src/repositories/identityMetadata.ts

```diff
@@ -101,9 +101,9 @@ export class IdentityMetadataRepository {
     metadata: RawMetadataState
   ): metadata is MetadataMapV2 => {
     return (
-      this.rawMetadata !== "loading" &&
-      this.rawMetadata !== "error" &&
-      this.rawMetadata !== "not-loaded"
+      metadata !== "loading" &&
```

Comment:
It works anyway, but I feld that it was better to do the checks on the passed parameter.

I needed to pass it so that I could do the type guard `metadata is MetadataMapV2`.